docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     ports:
       - "4000:4000"
     restart: always
-    command: npm run dev
+    command: pm2-runtime server.js -- --max-old-space-size=4096 NODE_ENV=production --trace-warnings
```

Comment:
I made some changes to devops flow. This change is not needed.

Reply:
sure, I'm going to revers this change